crates/ruff/src/rules/flake8_simplify/rules/ast_if.rs

```diff
@@ -65,6 +66,37 @@ impl Violation for ReturnBoolConditionDirectly {
     }
 }
 
+define_violation!(
+    /// ### What it does
+    /// Checks for three or more consective if-statements with direct returns
+    ///
+    /// ### Why is this bad?
+    /// These can be simplified by using a dictionary
+    ///
+    /// ### Example
+    /// ```if x = 1:
```

Comment:
```suggestion
    /// ```python
    /// if x = 1:
```